EDDiscovery/Forms/DownloadManagerForm.Designer.cs

```diff
@@ -17,6 +17,7 @@ protected override void Dispose(bool disposing)
             {
                 components.Dispose();
             }
+            font?.Dispose();
```

Comment:
Should `Dispose(bool disposing)` be moved out of the designer code, so the designer doesn't try to rewrite it?

Also, the font should only be disposed if `disposing` is true, as if `disposing` is false the `Dispose` method is being called by the finalizer, and it's not safe to directly dispose of the font (which would probably itself be being finalized).